src/components/CampusMap/EventPin.tsx

```
@@ -0,0 +1,61 @@
+  const count = group.events.length;
+  const emojiSrc = CATEGORY_EMOJI_SRC[lead.category];
+  const color = CATEGORY_COLOR[lead.category];
+  const dateLabel = parseEventDate(lead.date).toLocaleDateString(locale, { day: 'numeric', month: 'short' });
+
+  return (
+    <button
+      className="group pointer-events-auto absolute left-0 top-0 flex items-center justify-center leaflet-zoom-animated"
+      style={{ transform: `translate(${x}px, ${y}px) translate(-50%, -50%)` }}
+      title={lead.title}
+      onClick={() => onSelect(lead.id)}
+    >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)